webapp/src/app/bcviewer/show-block.html

```diff
@@ -27,7 +31,7 @@ <h1>Block {{data.sb.index}}</h1>
                         </div>
                         <div fxFlex="1 1 20em">ForwardLinks:
                             <span *ngFor="let forward of data.forwardLinks" (click)="goBlock(forward)" class="link">
-                                {{forward.index}}</span>
+                                {{forward.index}}/{{forward.sign}}</span>
```

Comment:
Maybe disable it if `forward.sign === undefined`?

Reply:
Is there a code-path where this can happen? All forward-links should have a `sign` field.

Reply:
Correct, it will always be defined; for the empty string case, it will show as `$index/`, it can be weird to have a trailing slash, but that's a matter of taste.

Reply:
well, not as weird as having a forward link without signatures!